package.json

```diff
@@ -1,27 +1,23 @@
 {
     "name": "@gridsuite/commons-ui",
-    "version": "0.43.0",
+    "version": "0.44.0",
```

Comment:
should we update commons-ui version in a specific PR as usual ?

Reply:
Yes mb